hal/lib/athena/HAL.cpp

```diff
@@ -7,11 +7,11 @@
 
 #include "HAL/HAL.h"
 
-#include <signal.h>  // linux for kill
-#include <stdlib.h>
 #include <sys/prctl.h>
 #include <unistd.h>
 
+#include <csignal>  // linux for kill
```

Comment:
kill() is only defined in signal.h on POSIX systems, but the C standard library functions are in csignal. We could either use signal.h for both or figure out a smarter way to resolve the ambiguity between signal.h and csignal.